### PR TITLE
TASK: Move the DimensionSwitcher to the PrimaryToolbar

### PR DESCRIPTION
**Pull request by markusguenther**
Thanks to @dfeyer who did the initial PR for that. As it was pretty outdated I created a new branch upon master and cherry picked the changes from @dfeyer.

Actually used the code, get rid of the middle section and put the switcher to the right.
Also adjusted the styles a bit to fit the look and feel of the other drop downs in the top bar.

### CODE REVIEW

**Comment by markusguenther** [comment]
Bildschirmfoto 2019-11-25 um 20 05 44

I really don`t get the why this error happens on circleCI. Locally this works, but therefore locally (on mac os catalina) the master has errors when changing the page name in the inspector.

That does not happen on circleCI as well :(

**Comment by markusguenther** [comment]
Bildschirmfoto 2019-11-25 um 20 34 06
Ok for some reason the drop down is gone on the saucelabs circleCI test.

**Comment by markusguenther** [comment]
Ok problem is that we now have to much in the top bar and the dropdown is really not visible because the bar is not scrollable.

File: Tests/IntegrationTests/e2e.sh [hunk 1]
```diff
@@ -34,6 +34,7 @@ for fixture in Packages/Application/Neos.Neos.Ui/Tests/IntegrationTests/Fixtures
             --selector-timeout=10000 --assertion-timeout=30000
     cd ../../..
     rm -f DistributionPackages/Neos.TestSite
+    rm -f DistributionPackages/Neos.TestNodeTypes
```
Comment (dimaip) [inline Tests/IntegrationTests/e2e.sh]: What's this?

File: Tests/IntegrationTests/SharedNodeTypesPackage/SharedNodeTypesPackage [hunk 2]
```diff
@@ -0,0 +1 @@
+../Packages/Application/Neos.Neos.Ui/Tests/IntegrationTests/SharedNodeTypesPackage
```
Comment (dimaip) [inline Tests/IntegrationTests/SharedNodeTypesPackage/SharedNodeTypesPackage]: What's this?

File: packages/neos-ui/src/Containers/PrimaryToolbar/DimensionSwitcher/style.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    padding-top: var(--spacing-Half);
+    padding-bottom: var(--spacing-Quarter);
+}
+
+.buttonGroup {
+    text-align: right;
+    margin-top: var(--spacing-Half);
+}
+
+@media only screen and (max-width: 1024px) {
```
Comment (dimaip) [inline packages/neos-ui/src/Containers/PrimaryToolbar/DimensionSwitcher/style.css]: Ideally I'd do responsive stuff right in react, as some sort of component like `<MediumDown>something</MediumDown>`.
But feel free to leave it as is for this change.

File: Tests/IntegrationTests/SharedNodeTypesPackage/SharedNodeTypesPackage (hunk 2 above)
Comment (markusguenther) [reply Tests/IntegrationTests/SharedNodeTypesPackage/SharedNodeTypesPackage]: That is maybe the reason for the  note in the next point... did not see that :/
Just wondered that the symlink could not be created. Therefore (rm - rf DistributionPackages/Neos.TestNodeTypes)

**Comment by markusguenther** [comment]
@dimaip thanks for the review... maybe it was to late yesterday and totally missed the commited symlink :(

File: packages/neos-ui/src/Containers/PrimaryToolbar/DimensionSwitcher/style.css (hunk 3 above)
Comment (markusguenther) [reply packages/neos-ui/src/Containers/PrimaryToolbar/DimensionSwitcher/style.css]: Personally like to have such things in CSS, but sometimes it is maybe easier to hide the content via react.

Comment (dimaip) [reply packages/neos-ui/src/Containers/PrimaryToolbar/DimensionSwitcher/style.css]: Haha, I wish there was no CSS in neos-ui (styled-components & co) :)

**Comment by dimaip** [comment]
Np, that's what reviews are for :)

**Comment by markusguenther** [comment]
Thanks Dmitri 🙇

**Pull request merged**